### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by peteruithoven**
Tried making the TimeManager code more readable

### CODE REVIEW

File: src/Services/TimeManager.vala [hunk 1]
```diff
@@ -25,28 +25,23 @@ interface Manager : Object {
...
 
     public TimeManager () {
-        update_current_time ();
-
-        if (current_time == null) {
-            return;
-        }
-
-        add_timeout ();
+        tick();
```
Comment (danrabbit) [inline src/Services/TimeManager.vala]: I don't think this is a good method name. `update_current_time` was better imo

Comment (peteruithoven) [reply src/Services/TimeManager.vala]: But... I didn't change that name. I've renamed `add_timeout` to `tick` and this function does what should happen every "tick": update time, call the signal, set timeout to be called again. 
One of my issues was that there where already 2 places that had to call: `update_current_time`, `minute_changed` and `add_timeout`.

Comment (danrabbit) [reply src/Services/TimeManager.vala]: Er yeah, I guess my point is that `tick` doesn't tell me anything about what this method does. I couldn't even tell what it was replacing :p

**Comment by peteruithoven** [comment]
Alright, giving up on this one, waste of our time.

**Pull request closed without merging**